tools/releaseBuild/azureDevOps/templates/upload-final-results.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
+  artifactPath:
+  artifactFilter: '*'
+  condition: succeeded()
```

Comment:
Maybe add a parameter for container name to make this task more general and default it to `finalResults`

Reply:
This change can be made later.